### DISCUSSION TITLE
Why my commits on private repositories dont apear?

### DISCUSSION THREAD

**Discussion by diegomarty** [community/community#56096] [Repositories] [answered]
### Select Topic Area

Question

### Body

I was working on private repositories this weekend, and if I remember correctly, when I made a commit it was registered in my profile panel, but in this case nothing appears, does anyone know why?

**Comment by ichenglin** [comment, marked as answer]
Have you enabled **`Include private contributions on my profile`** option under **Contributions & Activity** in **[profile settings](https://github.com/settings/profile)**?

**Reply by diegomarty** [reply]
Sorry, I found out what was happening, there was an error in the email assigned to comments, sorry for the inconvenience and thank you very much!
